docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>Please add a description about the logic used to select these qubits. Are we using the current calibration values from heatmap? Or choosing them randomly? Also, we are choosing the multiple line segments in increasing order of length - any special reason why? </p><p> </p><p> </p><p>Let's also add a link to the calibration metrics tutorial similar to "Select Qubits" section in</p><p>https://github.com/quantumlib/Cirq/blob/master/docs/tutorials/google/xeb_calibration_example.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Outdated, but new logic on qubit selection for parallel XEB is just "select the entire chip"!</p>